### PR TITLE
Show beta watermark in gisf

### PR DESCRIPTION
**Pull request by Lukewh**

### CODE REVIEW

**Comment by Lukewh** [comment]
QA in gisf you should get the following:

screen shot 2017-02-17 at 15 29 40

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6808/

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -2903,6 +2903,10 @@ YUI.add('juju-gui', function(Y) {
       if (!gui || (gui && !gui.inspector)) {
         this._renderAddedServices();
       }
+
+      if (this.get('gisf')) {
```
Comment (hatched) [inline jujugui/static/gui/src/app/app.js]: This method gets called every time the db updates - We don't need to render this every time the db updates.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6809/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6810/

**Comment by hatched** [comment]
QA OK

When an application icon is under the Beta sign you can't drag it without highlighting all the text. I would normally say to change this to a normal element with pointer events turned off but I figure since this is short lived it'll be ok to leave it as is.

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**